DSA.html

```diff
@@ -116,11 +116,8 @@
 
 
 
-    <iframe title="DSA" style="border:none" width="1400" height="635"
+    <iframe title="DSA" style="border:none" width="100%" height="600vh"
         src="https://whimsical.com/embed/L7xNuV2ZRSsLd6vrVtyvrQ"></iframe>
-
-    <button class="button1"><a href="index.html">Go Back </a></button>
```

Comment:
Why is the Go back button removed?

Reply:
@harikesh409 go back Button is linked to index.html and the page has nav bar so the user can use it to go to the home page.

Reply:
okay